### PR TITLE
checking for AnonymousUser to prevent 500 errors

### PR DESCRIPTION
**Pull request by chavesj**
[Enterprise Jira Ticket](https://openedx.atlassian.net/browse/ENT-2728)

URL used to reproduce in stage:
https://enterprise-catalog.stage.edx.org/api/v1/enterprise-catalogs/ead5dbb2-096f-4c73-832f-5fd37e67849c/contains_content_items/

Link to logs in Splunk:
https://splunk.edx.org/en-US/app/search/search?sid=1586815750.659067

### CODE REVIEW

File: edx_rbac/utils.py [hunk 1]
```diff
@@ -63,6 +63,8 @@ def user_has_access_via_database(user, role_name, role_assignment_class, context
     The context argument is evaluated against the context(s) received from the role_assignment_class while accounting
     for the ALL_ACCESS_CONTEXT to grant access.
     """
+    if user.is_anonymous:
```
Comment (adamstankiewicz) [inline edx_rbac/utils.py:66]: Perhaps `if hasattr(user, 'is_anonymous') and user.is_anonymous:`, in case the `user` argument passed in is `None`?

Comment (iloveagent57) [reply edx_rbac/utils.py:66]: ...or `getattr(user, 'is_anonymous', False)`, if we're doing code golf :)

Comment (adamstankiewicz) [reply edx_rbac/utils.py:66]: That, too haha. Basically, just want to be sure we prevent an error from occurring when trying to access a non-existent attribute on a `None` type 😄

Comment (chavesj) [reply edx_rbac/utils.py]: ✅

**Comment by adamstankiewicz** [comment]
@chavesj Just needs the version bump and CHANGELOG entry 👍

**Pull request merged**